mopidy/core/actor.py

```diff
@@ -130,37 +132,37 @@ def state_changed(
             self.playback.set_state(new_state)
             self.playback._trigger_track_playback_paused()
 
-    def playlists_loaded(self):
+    def playlists_loaded(self) -> None:
         # Forward event from backend to frontends
         CoreListener.send("playlists_loaded")
 
-    def volume_changed(self, volume):
+    def volume_changed(self, volume: int) -> None:
```

Comment:
I'm uncertain if I want to go with "Percentage" to communicate the range of the value or "Volume" to avoid mixing this int with other ints flying around, leaning a bit towards the latter, if anything at all. What do you think?

Reply:
I'd prefer percentage so I have the hint that it's in the range 0 to 100. Otherwise I find the term voltage a bit vague and not much of a hint. But maybe I'm missing the point of the typing there.

Reply:
I've applied Percentage and DurationMs to the Core API as well as the Backend API and the Audio API where the values was passed downward in the stack now.